app/views/shared/_paginator.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="paginator">
+  <ul class="pagination">
+    <li class="<%= previous_status %>">
+      <%= link_to bets_path(page: '1'), class: 'page-link' do %>
+        <i class="fa fa-angle-double-left"></i>
+      <% end %>
+    </li>
+    <li class="<%= previous_status %>">
+      <%= link_to bets_path(page: current_page.to_i - 1), class: 'page-link' do %>
+        <i class="fa fa-angle-left"></i>
+      <% end %>
+    </li>
+
+    <% blocks.each do |i|%>
+      <% status = '' %>
```

Comment:
Technically this is not needed. Unlike some other languages, in ruby a variable initalized within an if block will be available outside the if statement, even if it doesnt run.

```ruby
if 1 == 3
  thing = 6
end 

puts thing.inspect # => nil
```

Also, since this is just one big block of ruby, you dont need to wrap each line in `<% %>` 

you could write it:
 
```ruby
<%
  if stuff 
    ...
  end
%>
```